### PR TITLE
Feature/ch13

### PR DESCRIPTION
**Pull request by monkeywithbanana**

### CODE REVIEW

File: 13/Node.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	{
+		return mNext;
+	}
+
+	int Node::GetData() const
+	{
+		return mData;
+	}
+
+	string Node::GetKey() const
```
Comment (popekim) [inline 13/Node.cpp]: return const string&

File: 13/SimpleHashMap.cpp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+				while (current != nullptr)
+				{
+					cout << "Key: " << current->GetKey() << ", Value: " << current->GetData() << endl;
+					current = current->GetNext();
+				}
+			}
+		}
+	}
+
+	/*constexpr int SimpleHashMap::hashFunctionWithCompileError(const char* s)
```
Comment (popekim) [inline 13/SimpleHashMap.cpp:87]: you can remove it if u want

File: 13/SimpleHashMap.h [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	public:
+		SimpleHashMap() = default;
+		~SimpleHashMap() = default;
+
+		void Add(const char* key, int value);
+		int Get(const char* key) const;
+		void Print() const;
+
+	private:
+		//static constexpr int hashFunctionWithCompileError(const char* s);
```
Comment (popekim) [inline 13/SimpleHashMap.h:22]: same

**Review by popekim — APPROVED** [review]
merge after fix

File: 13/SimpleHashMap.cpp (hunk 2 above)
Comment (monkeywithbanana) [reply 13/SimpleHashMap.cpp:87]: I was going to show this in the slide and say something like
`But this doesn't work while the other does`

**Pull request merged**